recipes/clickhouse-cityhash/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
```

Comment:
Please consider using jinja templating variables as in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L3-L19

Reply:
@synapticarbors like this?